recipes/nmrml2isa/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: nmrml2isa
+  version: "0.3.0"
+
+source:
+  fn: nmrml2isa-0.3.0.tar.gz
+  url: https://pypi.python.org/packages/61/87/eb7ed3b5dfcdaa5109355daf7ccab722ce984cf49283bead437d6a51cd8a/nmrml2isa-0.3.0.tar.gz
+  md5: 3f56e532da5ab8d7ee62bd32c1d21253
+
+build:
+  skip: True # [py26 and py3k]
+  number: 0
+  entry_points:
+    - nmrml2isa = nmrml2isa.parsing:run
+
+requirements:
+  build:
+    - python >=2.7,<3
```

Comment:
Constraining Python like this leads to conda not pinning to the minor Python version. This usually leads to problems later on. The canonical way to avoid this, is to just use build-skip and keep Python in run and build requirements unconstrained. Can you also fix your mzml2isa recipe in this way, please?